### PR TITLE
Feature/Multiprocessing

### PR DESCRIPTION
**Pull request by asajatovic**
- Enable multiprocessing support for [`Language.pipe()`](https://spacy.io/api/language#pipe)
- Move general tests to single file `test_spacy_udpipe.py`

### CODE REVIEW

**Review by FilipBolt — COMMENTED** [review]
LGTM, but please update this CR after the first one is pushed to make sure I don't miss anything, then I'll review again.

**Comment by asajatovic** [comment]
> LGTM, but please update this CR after the first one is pushed to make sure I don't miss anything, then I'll review again.

Ofc :)

File: spacy_udpipe/udpipe.py [hunk 1]
```diff
@@ -40,11 +42,50 @@ def nextSentence(self, sentence: Sentence, _: ProcessingError) -> bool:
...
+        self.path = path
+        self._model = super().load(path)
+        if self._model is None:
+            raise Exception(f"Cannot load UDPipe model from file '{path}'")
+
+    def __reduce__(self):
+        # pickle support
+        return (self.__class__, (self.path,))
+
+    @property
```
Comment (asajatovic) [inline spacy_udpipe/udpipe.py:63]: The `@property` decorator is used as all of the `_Model` attributes are read-only.

File: tests/test_spacy_udpipe.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+def lang() -> str:
+    return EN
+
+
+@pytest.fixture(autouse=True)
+def download_lang(lang: str) -> None:
+    download(lang=lang)
+
+
+def test_serialization(lang: str) -> None:
```
Comment (FilipBolt) [inline tests/test_spacy_udpipe.py:23]: I assume this requires picking support?

File: spacy_udpipe/udpipe.py [hunk 3]
```diff
@@ -40,11 +42,50 @@ def nextSentence(self, sentence: Sentence, _: ProcessingError) -> bool:
...
+
+    def __init__(self, path: str):
+        self.path = path
+        self._model = super().load(path)
+        if self._model is None:
+            raise Exception(f"Cannot load UDPipe model from file '{path}'")
+
+    def __reduce__(self):
+        # pickle support
+        return (self.__class__, (self.path,))
```
Comment (FilipBolt) [inline spacy_udpipe/udpipe.py:61]: great :)

File: tests/test_spacy_udpipe.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    nlp = load(lang=lang)
+    nlp.to_disk("./udpipe-spacy-model")
```
Comment (FilipBolt) [inline tests/test_spacy_udpipe.py]: nitpick, you could use a tempfile for this

**Review by FilipBolt — APPROVED** [review]
LGTM, minor non-blocking comments

Already at feature 0.3.0, very nice!

File: tests/test_spacy_udpipe.py (hunk 2 above)
Comment (asajatovic) [reply tests/test_spacy_udpipe.py:23]: Actually, it worked fine without it (`test_serialization` was just relocated).
`Language.pipe()` did not work without `pickle` support.

**Pull request merged**